ts/session/utils/String.ts

```diff
@@ -63,5 +63,9 @@ export const sanitizeSessionUsername = (inputName: string) => {
   return validChars;
 };
 
-export const ed25519Str = (ed25519Key: string) =>
-  `(...${ed25519Key.length > 58 ? ed25519Key.substr(58) : ed25519Key})`;
+export const ed25519Str = (ed25519Key: string) => {
+  if (ed25519Key.length <= 6) {
+    return `(${ed25519Key})`;
+  }
+  return `(${ed25519Key.substr(0, 2)}...${ed25519Key.substr(-4)})`;
+};
```

Comment:
This kind of exists already using the StringCollapser in another branch. Can you model this function on this snippet? It follows what we do on the websites.

```tsx
export const StringCollapser = ({
  str,
  leadingChars = 6,
  trailingChars = 4,
  separtorStyle,
}: {
  str: string;
  leadingChars?: number;
  trailingChars?: number;
  separtorStyle?: CSSProperties;
}) => {
  if (str.length <= leadingChars + trailingChars + 3) {
    return <>{str}</>;
  }

  return (
    <span>
      {str.slice(0, leadingChars)}
      <span style={separtorStyle}>…</span>
      {str.slice(-trailingChars)}
    </span>
  );
};
```

Reply:
Maybe `collapseString()`?